docs/service/webapp.rst

```diff
@@ -23,21 +23,6 @@ To ease debugging, you can run a shell in the container::
 Then you can start and stop the webapp, adjust files, and debug.
 
 
-.. Note::
-
-   The ``STATIC_ROOT`` is set to ``/tmp/crashstats-static/`` rather than
-   ``/app/webapp-django/static``. This alleviates permissions-related problems
-   because the process in the container runs as uid 10001 which is not the uid
-   of the user you're using on your host computer.
-
-   The problem this creates is that ``/tmp/crashstats-static/`` is ephemeral
-   and any changes there disappear when you stop the container.
-
-   If you want it persisted, you should mount that directory using ``volumes``
-   in a ``docker-compose.override.yml`` file.
-
-   https://docs.docker.com/compose/extends/
-
```

Comment:
This is still true. It's something I keep forgetting to look into, but it's still the case. The `STATIC_ROOT` for the local dev environment is set in `docker/config/never_on_a_server.env`.

I can explain what's going on here, but it's sort of complicated and requires copious hand-waving.

Reply:
Oh, my bad--you just moved it.